ImageFactoryV3-Build.ps1

```diff
@@ -445,8 +472,8 @@ Invoke-Command -ComputerName $($Settings.Settings.HyperV.Computername) -ScriptBl
             VMHost = $($property.VMHost.'#text');
             VMName = $($property.VMName.'#text');
             LastTime = $($property.LastTime.'#text') -replace "T"," ";
-            StartTime = $($property.StartTime.�#text�) -replace "T"," "; 
-            EndTime = $($property.EndTime.�#text�) -replace "T"," "; 
+            StartTime = $($property.StartTime.'#text') -replace "T"," "; 
```

Comment:
There where some strange character in there which some editors (VS Code for example) didn't like so much...